addons/mail/static/tests/owl/component/discuss_tests.js

```diff
@@ -3739,6 +3743,115 @@ QUnit.test('composer: add an attachment', async function (assert) {
         "should have an attachment remove button");
 });
 
+QUnit.test('composer: drop attachments', async function (assert) {
```

Comment:
Would be nice to have composer_tests.js, since tests focus on composer

Reply:
Maybe yes, but I don't know what I should do in the test component initialization ?